src/app/components/Header/HeaderSearchBar/HeaderSearchBar.tsx

```diff
@@ -36,7 +36,10 @@ export const HeaderSearchBar = ({
 	const debouncedQuery = useDebounce(query, debounceTimeout);
 	useEffect(() => onSearch?.(debouncedQuery), [onSearch, debouncedQuery]);
 
-	const handleQueryReset = () => {
+	const handleQueryReset = (event: React.MouseEvent<HTMLButtonElement>) => {
+		event.preventDefault();
+		event.stopPropagation();
+
```

Comment:
There's no need for this if you add `type="button"` to the reset button below.

Reply:
Guilty. Had eyes on several PRs and went straight to change the handler. @dated  patched